src/Keycloak.Net.Core/Common/Converters/EnumMemberJsonStringEnumConverter.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.Json;
+
+namespace Keycloak.Net.Common.Converters;
+
+/// <summary>
+/// Provides an AOT-compatible extension for <see cref="JsonStringEnumConverter"/> that adds support for <see cref="EnumMemberAttribute"/>.
+/// </summary>
+/// <typeparam name="TEnum">The type of the <see cref="TEnum"/>.</typeparam>
+public sealed class EnumMemberJsonStringEnumConverter<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] TEnum>()
```

Comment:
@CesarD I'm not super familiar with AOT, does this need to be referenced anywhere in the code, or does it work automatically.

I guess I am asking whether `[JsonConverter(typeof(JsonStringEnumConverter))]` is sufficient, or if it should be `[JsonConverter(typeof(EnumMemberJsonStringEnumConverter))]`?

Reply:
Never mind, I just found it in use here in the Category class, so it wasn't used as I expected

Reply:
@CesarD great, thanks. I had assumed that it was an "extension" (of sorts) to make `JsonStringEnumConverter` work with AOT, but when I saw it in use I worked out what it was doing.